docker-compose.sonar.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
```

Comment:
Wouldn't it be better to have these in the separate directory ?

Reply:
I don't mind to have it here. But we can change the location if you want. Where do you place it?

Reply:
I'd rather place all of the functionality into the single directory, maybe `scripts/sonarqube`. This is not something we use daily, so no need to spread it around through the whole codebase.

Reply:
There is already /dev/sonar directory. So I will place docker-compose file there.